prometheus-to-sd/translator/translator.go

```diff
@@ -123,11 +125,69 @@ func translateOne(config *config.GceConfig, component string, name string, mType
 			Type:   "gke_container",
 		},
 		MetricKind: metricKind,
-		ValueType:  "INT64",
+		ValueType:  valueType,
 		Points:     []*v3.Point{point},
 	}
 }
 
+func setValue(mType dto.MetricType, metric *dto.Metric, point *v3.Point) {
+	if mType == dto.MetricType_GAUGE {
+		val := int64(metric.GetGauge().GetValue())
+		point.Value.Int64Value = &val
+		point.ForceSendFields = append(point.ForceSendFields, "Int64Value")
+	} else if mType == dto.MetricType_HISTOGRAM {
+		point.Value.DistributionValue = getHistogramValue(metric.GetHistogram())
+		point.ForceSendFields = append(point.ForceSendFields, "DistributionValue")
+	} else {
+		val := int64(metric.GetCounter().GetValue())
+		point.Value.Int64Value = &val
+		point.ForceSendFields = append(point.ForceSendFields, "Int64Value")
+	}
+}
+
+func getHistogramValue(h *dto.Histogram) *v3.Distribution {
+	count := int64(h.GetSampleCount())
+	mean := float64(0)
+	dev := float64(0)
+	buckets := []float64{}
```

Comment:
this is very misleading name - please rename to 'bounds' or 'bucketBorders' or something more explicit.